app/src/main/java/com/bunbeauty/papakarlo/feature/consumercart/ConsumerCartFragment.kt

```diff
@@ -38,145 +38,236 @@ import com.bunbeauty.papakarlo.common.ui.screen.ErrorScreen
 import com.bunbeauty.papakarlo.common.ui.screen.LoadingScreen
 import com.bunbeauty.papakarlo.common.ui.theme.FoodDeliveryTheme
 import com.bunbeauty.papakarlo.common.ui.theme.bold
+import com.bunbeauty.papakarlo.common.ui.theme.medium
 import com.bunbeauty.papakarlo.databinding.LayoutComposeBinding
-import com.bunbeauty.papakarlo.extensions.setContentWithTheme
 import com.bunbeauty.papakarlo.feature.consumercart.ConsumerCartFragmentDirections.toCreateOrderFragment
 import com.bunbeauty.papakarlo.feature.consumercart.ConsumerCartFragmentDirections.toLoginFragment
 import com.bunbeauty.papakarlo.feature.consumercart.ConsumerCartFragmentDirections.toMenuFragment
 import com.bunbeauty.papakarlo.feature.consumercart.ConsumerCartFragmentDirections.toProductFragment
 import com.bunbeauty.papakarlo.feature.consumercart.ui.CartProductItem
+import com.bunbeauty.papakarlo.feature.menu.ui.MenuProductItem
 import com.bunbeauty.shared.domain.model.SuccessLoginDirection
 import com.bunbeauty.shared.presentation.consumercart.CartProductItem
 import com.bunbeauty.shared.presentation.consumercart.ConsumerCartData
-import com.bunbeauty.shared.presentation.consumercart.ConsumerCartEvent
-import com.bunbeauty.shared.presentation.consumercart.ConsumerCartUIState
+import com.bunbeauty.shared.presentation.consumercart.ConsumerCartState
 import com.bunbeauty.shared.presentation.consumercart.ConsumerCartViewModel
 import org.koin.androidx.viewmodel.ext.android.viewModel
 
-class ConsumerCartFragment : BaseFragmentWithSharedViewModel(R.layout.layout_compose) {
+class ConsumerCartFragment :
+    BaseComposeFragment<ConsumerCartState.State, ConsumerCartState.Action, ConsumerCartState.Event>() {
```

Comment:
Lets rename `ConsumerCartState` to `ConsumerCart`

Reply:
We have same name for another model
Can i rename ConsumerCart in domain to ConsumerCartDomain?